msg/ZFS-8000-ER/index.html

```diff
@@ -225,4 +225,91 @@
 all pools are healthy
 </pre>
 </td></tr></table>
+<br><br>
+<hr>
+<br><br>
+<!-- end: templates/success -->
+<!-- start: article -->
+<!-- code: ZFS-8000-ER -->
+<!-- keys: errata -->
+<table width="80%"><tr><td>
+<a name="4"><b>ZFS Errata #4</b></a>
+<p>
+<dl>
+<dt><p><b>Type</b>
+<dd>Compatibility
+<dt><p><b>Severity</b>
+<dd>Moderate
+<dt><p><b>Description</b>
+<dd>An encrypted dataset contains an on-disk format incompatibility.
+<dt><p><b>Automated Response</b>
+<dd>No automated response will be taken.
+<dt><p><b>Impact</b>
+<dd>
+Encrypted datasets created before the ZFS packages were updated cannot be
+backed up via a raw send to an updated system. These datasets also cannot
+receive additional snapshots.
+</dd>
+<dt><p><b>Suggested Action for System Administrator</b>
+<dd>
+First, system administrators with affected pools will need to enable the
+<b>bookmark_v2</b> feature on their pools. This will disable read-write
+and read-only compatibility with previous versions of the ZFS software.
```

Comment:
How about `Enabling this feature will prevent this pool from being imported by previous versions of the ZFS software (including read-only imports).`